rollup.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  input: "main.js", // Entry file
```

Comment:
let's move the `main.js` --> `lib/cli.js`
and change the path here too.

Reply:
done